src/components/Comment/Comment.tsx

```diff
@@ -31,6 +32,7 @@ const commentControlsButton = (pillar: Pillar) => css`
   ${textSans.xsmall({ fontWeight: "bold" })}
   margin-right: ${space[2]}px;
   color: ${palette[pillar][400]};
```

Comment:
Is it worth fixes these as we go along?

Reply:
prob best to have its own PR for most of these issues